sdk/storage/azure-storage-blob/tests/test_retry_async.py

```diff
@@ -527,6 +530,38 @@ async def test_streaming_retry(self, **kwargs):
             count = [0]
             blob._pipeline._transport.send = self._count_wrapper(count, blob._pipeline._transport.send)
             await blob.download_blob()
-        assert stream_reader_read_mock.call_count == count[0] == 4
+        assert stream_reader_read_mock.call_count == count[0] == 12
```

Comment:
This has now become 12 after popping back in our manual retry logic (as we theorized previously). 
3 retries + 1 initial call from aiohttp retry policy
x
3 retries from manual policy
--> 12